test/functional/app/api/test_ussd_api.py

```diff
@@ -131,11 +114,11 @@ def mock_send_message(phone, message):
 
 
 def test_invalid_service_code(mocker, test_client,
-                                init_database, initialised_blockchain_network, init_seed):
+                              init_database, initialised_blockchain_network, init_seed):
 
     org = OrganisationFactory()
     sender = UserFactory(preferred_language="en", phone=make_kenyan_phone(phone()), first_name="Bob", last_name="Foo",
-        pin_hash=User.salt_hash_secret('0000'), default_organisation=org)
+                         pin_hash=User.salt_hash_secret('0000'), default_organisation=org)
```

Comment:
please args below each other or just one indent

Reply:
The [`self-service-wallets-func-tests`](https://github.com/GrassrootsEconomics/SempoBlockchain/blob/philip/self-service-wallets-func-tests/test/functional/app/api/test_ussd_api.py) PR cleans all these lint issues.